connector/countconnector/factory_test.go

```diff
@@ -17,19 +17,32 @@ package countconnector
 import (
 	"testing"
 
-	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/component/componenttest"
 )
 
 func TestFactory_Type(t *testing.T) {
 	factory := NewFactory()
-	assert.Equal(t, factory.Type(), component.Type(typeStr))
+	require.Equal(t, factory.Type(), component.Type(typeStr))
 }
 
 func TestFactory_CreateDefaultConfig(t *testing.T) {
 	factory := NewFactory()
 	cfg := factory.CreateDefaultConfig()
-	assert.Equal(t, cfg, &Config{})
-	assert.NoError(t, componenttest.CheckConfigStruct(cfg))
+	require.Equal(t, cfg, &Config{
```

Comment:
Not for this PR, but shouldn't this be an assert? `require` is to ensure the test requirements are valid, whereas `assert` is to validate the results of the test.

Reply:
True, I need to do better using these appropriately. I need to rebase anyways so I'll take a pass through these tests.